sdk/src/test/scala/com/horizen/fixtures/SecretFixture.scala

```diff
@@ -126,7 +126,8 @@ trait SecretFixture {
     val payload = "This is string to sign"
     val message = payload.getBytes(StandardCharsets.UTF_8)
     val pair = Keys.createEcKeyPair
-    new SignatureSecp256k1(Sign.signMessage(message, pair, true))
+    val signatureData = Sign.signMessage(message, pair, true)
+    new SignatureSecp256k1(signatureData.getV, signatureData.getR, signatureData.getS)
```

Comment:
let's use `Secp256k1.sign` method here and in all other similar places in tests.
The goal is to avoid using `Sign.signMessage(message, pair, true)` directly as well as in the main code.